src/category_theory/limits/fubini.lean

```diff
@@ -0,0 +1,243 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import category_theory.limits.limits
+import category_theory.products.basic
+import category_theory.currying
+
+/-!
+# A Fubini theorem for limits
```

Comment:
```suggestion
# A Fubini theorem for categorical limits
```
Just to quickly help the poor person who finds a `fubini.lean` through any sort of search tool, and doesn't pay attention to the directory names...